src/api/controllers/articlesController.js

```diff
@@ -0,0 +1,66 @@
+import model from '../models/index';
+import status from '../../helpers/constants/status.codes';
+import errorMessage from '../../helpers/constants/error.messages';
+import errorSender from '../../helpers/error.sender';
+/**
+ * @class
+ */
+export default class Articles {
+  /**
+   * This function gets all the articles that are stored in the article table
+   *
+   * @static
+   * @param {*} req the request
+   * @param {*} res the response to be sent
+   * @memberof Articles
+   * @returns {Object} res
+   */
+  static async getArticles(req, res) {
+    const { Article } = model;
+    const { page } = req.query;
+    const articles = await Article.findAndCountAll();
+    const limit = 10;
```

Comment:
@truestbyheart Limit and offset should be changeable, this means we shouldn't hardcode their values in the backend. You should accept limit and offset from req.query also, and set then defaults eng

```
limit = req.query || 10
```

Reply:
Okay,